packages/babel-standalone/src/preset-stage-2.js

```diff
@@ -0,0 +1,22 @@
+import presetStage3 from "./preset-stage-3";
+
+import transformDecorators from "@babel/plugin-proposal-decorators";
+import transformFunctionSent from "@babel/plugin-proposal-function-sent";
+import transformExportNamespaceFrom from "@babel/plugin-proposal-export-namespace-from";
+import transformNumericSeparator from "@babel/plugin-proposal-numeric-separator";
+import transformThrowExpressions from "@babel/plugin-proposal-throw-expressions";
+
+export default (_, opts = {}) => {
+  const { loose = false, useBuiltIns = false, decoratorsLegacy = false } = opts;
+
+  return {
+    presets: [[presetStage3, { loose, useBuiltIns }]],
```

Comment:
`useBuiltIns` can be removed, since it was only used by object-rest-spread.
It can also be removed in the stage-0 and stage-1 presets.